.github/ISSUE_TEMPLATE/clarification-request.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Clarification request
+about: You’ve found something confusing
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  Please paste (a) link(s) from your browser showing the issue (bare
+  so it can be seen, not buried in a Markdown link).
+
+  Often a clarification request has as much to do with _where_ in the
+  docs something is addressed as _how._ If you think something is
+  useful but in the wrong place(s), or needs to be in multiple places
+  because of where different readers will look, say so!
+
+  If you're not sure where your request for clarification could be
+  best dealt with, either provide the URL where you think it should
+  be, provide multiple possible URLs, or delete the line.
+
+  Please don't provide a URL you wish existed that does not -- that
+  should be in a "New material" request instead.
+
+  -->
+  **Found in URL:** URL-HERE
+<!--
+  Also include the commit hash shown at the bottom of your browser
+  page, so we know what version you’re looking at. Better yet, include
+  the entire “From FILE.pod6 at perl6/doc on GitHub, commit a0b1c2d.”
+  line you’ll find as the first line of every doc page’s footer. -->
+**Found at commit:** COMMIT-OR-FROM-LINE-HERE
+## The problem
+
+<!--
+
+  Please describe the content you feel needs clarification. A quote,
+  cut and pasted, would be much appreciated (use `> ` or a blockquote
+  so we can easily see where it begins and ends).
+
+-->
+
+## Suggestions
+
+
+<!--
+  If applicable, tell us in "Suggestions" what could be done to solve
+  the problem, such as "Rephrase the description" or "Change the
+  example program so it doesn't introduce two unrelated concepts at
+  once."
+
+  Please note we do not have the ability to add diagrams at present
```

Comment:
Remove this. Just because we're unable doesn't mean that people shouldn't suggest to do that.

Reply:
Done.